sky/backends/backend_utils.py

```diff
@@ -151,6 +153,92 @@
 ]
 
 
+# TODO(tian): Refactor to controller_utils. Current blocker: circular import.
+@dataclasses.dataclass
+class _ControllerSpec:
+    """Spec for skypilot controllers."""
+    name: str
+    cluster_name: str
+    in_progress_hint: str
+    decline_cancel_hint: str
+    decline_down_in_init_status_hint: str
+    decline_down_for_dirty_controller_hint: str
+    check_cluster_name_hint: str
+    default_hint_if_non_existent: str
+
+
+class Controllers(enum.Enum):
```

Comment:
Why can't we keep this `Controllers` and `_ControllerSpec` in the `controller_utils`?

Reply:
Good point! I was still stuck in the old thinking where `controller_utils` imports `backend_utils`. Just move it back.